.github/workflows/ci.yml

```diff
@@ -2,14 +2,14 @@ name: Continuous Integration
 on:
   push:
     branches:
-    - master
+    - main
     tags:
     - '*'
   pull_request:
 
 env:
-  RASA_X_DEMO_VERSION: '0.33.0'
-  RASA_IMAGE_TAG: '2.1.3-full'
+  RASA_X_DEMO_VERSION: '0.35.0'
```

Comment:
had to bump this as it was not part of the page size anymore (increasing the page size didn't work). I think we need a better approach for this, what do you think?

Reply:
seems it's not used as the test runs against `main`?

Reply:
> had to bump this as it was not part of the page size anymore (increasing the page size didn't work). I think we need a better approach for this, what do you think?

Yes, I'll take care of it, We can use the Manifest API to check a tag.

It should run on a PR

Reply:
Mhm, weird. Then I don't understand why the CI keeps failing with the old version 🤔 But it's also late 🙈